app/views/examples/vehicle-check-answers.html

```diff
@@ -0,0 +1,105 @@
+{% extends "layout.html" %}
+
+{% block page_title %}
+  Check your answers
+{% endblock %}
+
+{% block content %}
+
+<main id="content" role="main">
+  
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <h1 class="heading-large">
+        Check your answers before sending your application
+      </h1>
+
+      <p>
+        When a user has entered data on pages, show it by using code like this:  
+      </p>
+
+      <pre>
```

Comment:
The markup for this section should be `<pre><code>...</code></pre>` we should also use the styles from `.code` for `pre` and `code` elements